cfme/tests/v2v/test_v2v_cancel_migrations.py

```diff
@@ -46,24 +45,19 @@ def cancel_migration_plan(appliance, provider, mapping_data_vm_obj_mini):
     )
     if migration_plan.wait_for_state("Started"):
         request_details_list = migration_plan.get_plan_vm_list(wait_for_migration=False)
-        vms = request_details_list.read()
-        # Random percentage to cancel migration at
-        cancel_migration_after_percent = 6
-
-        def _get_plan_status_and_cancel():
-            migration_plan_in_progress_tracker = []
-            for vm in vms:
-                clock_reading1 = request_details_list.get_clock(vm)
-                time.sleep(1)  # wait 1 sec to see if clock is ticking
-                if request_details_list.progress_percent(vm) > cancel_migration_after_percent:
-                    request_details_list.cancel_migration(vm, confirmed=True)
-                clock_reading2 = request_details_list.get_clock(vm)
-                migration_plan_in_progress_tracker.append(request_details_list.is_in_progress(vm)
-                                                          and (clock_reading1 < clock_reading2))
-            return not any(migration_plan_in_progress_tracker)
-
-        wait_for(func=_get_plan_status_and_cancel, message="migration plan is in progress,"
-                 "be patient please", delay=5, num_sec=3600)
+        vm_detail = request_details_list.read()[0]
+
+        def get_plan_status_and_cancel():
+            return request_details_list.is_in_progress(vm_detail)
+
+        wait_for(
+            func=get_plan_status_and_cancel,
+            delay=10,
+            num_sec=120,
+            message="migration plan is in progress, be patient please")
+
+        request_details_list.cancel_migration(vm_detail, confirmed=True)  # Cancel migration
+
     else:
         pytest.skip("Migration plan failed to start")
```

Comment:
The `skip` is probably not right in here. I would just made the the test go to ERROR state somehow. But I will let someone else to decide what to do.

Reply:
I think we agree with @mshriver that skip is not right here and should be changed to `pytest.fail` which if I understand correctly will make the test go to ERROR state. Please consider changing this as I think this is a small change that is easy to do. Not required for this PR though

Reply:
@jaryn good point here, I'm going to look for this update in a followup PR from @nachandr